pom.xml

```diff
@@ -8,7 +8,7 @@
   <artifactId>soot-shiftleft</artifactId>
   
   <name>Soot</name>
-  <version>0.2.34-SNAPSHOT</version>
+  <version>0.2.34</version>
```

Comment:
not sure about the exact release process, but are you going to release this very commit?

Reply:
I think after merge 0.2.34 would be released why?